packages/flutter/lib/src/animation/scroll_behavior.dart

```diff
@@ -105,26 +105,25 @@ class UnboundedBehavior extends ExtentScrollBehavior {
 Simulation _createFlingScrollSimulation(double position, double velocity, double minScrollOffset, double maxScrollOffset) {
   double startVelocity = velocity * _kSecondsPerMillisecond;
 
```

Comment:
All these "end" values should be prefixed with a comment about how if you have a transform scaling this stuff up, our thresholds are going to make no sense and it's going to get ugly.